decimal_test.go

```diff
@@ -15,8 +15,15 @@ import (
 	"testing"
 	"testing/quick"
 	"time"
+
+	"github.com/stretchr/testify/assert"
 )
 
+func init() {
+	// This is left for test backwards compability
+	TrimTrailingZeroes = true
```

Comment:
Not sure if we need, wouldn't it be better to replace the tests? Otherwise, I would suggest adding this to the `Main` test function that is called every time when the tests are started.

Reply:
I think we can move this `TrimTrailingZeroes` to the functions with the same comment so that we we can refactor in the future.